app/graphql/types/rock_and_pebble_type.rb

```diff
@@ -1,20 +1,20 @@
 module Types
-    class RockAndPebbleType < Types::BaseObject
-        field :id, ID, null: false
-        field :rock, Types::UserType, null: false
-        field :pebble, Types::UserType, null: false
-        field :active, Boolean, null: false
-        field :my_rocks, [Types::UserType], null: true
-        field :my_pebbles, [Types::UserType], null: true
+  class RockAndPebbleType < Types::BaseObject
+      field :id, ID, null: false
+      field :rock, Types::UserType, null: true
+      field :pebble, Types::UserType, null: true
```

Comment:
I think I'm catching up on this, can't remember what was said previously; why did this change?

Reply:
I don't think these changed. I believe we were discussing the my_rocks and my_pebbles fields. We decided to leave those in under the RockandPebble type in the end as well as having them as fields in the User type.

Reply:
Oh I know what happened - I commented these out while I was working, made a commit, and then put it back but it looks like the spacing is a bit different. I can go back and re-align them.

Reply:
Sorry, I meant the `null: true` bits. GitHub is showing that `null: false` was changed to `null: false` on lines 4 and 5 and I vaguely recall we talked about that at some point but I can't find the conversation.